client/backend.js

```diff
@@ -188,6 +188,10 @@ module.exports = {
     },
 
     updateAccess(accessId, access) {
+        if (access.customFields) {
+            access.customFields = JSON.stringify(access.customFields);
```

Comment:
The access object also (always) has a "password" property. I am not sure how you would use a ternary condition here ?